sdk/purview/azure-purview-account/README.md

```diff
@@ -0,0 +1,164 @@
+
+[Source code][source_code] |  [Package (PyPI)][account_pypi] | [API reference documentation][account_ref_docs]| [Product documentation][account_product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- Python 2.7, or 3.6 or later is required to use this package.
+- You must have an [Azure subscription][azure_subscription] and a [Purview][purview_resource] to use this package.
+
+#### Create a Purview Resource
+
+Follow [these][purview_resource] instructions to create your Purview resource
+
+### Install the package
+
+Install the Azure Purview Account client library for Python with [pip][pip]:
+
+```bash
+pip install azure-purview-account
+```
+
+### Authenticate the client
+
+To use an [Azure Active Directory (AAD) token credential][authenticate_with_token],
+provide an instance of the desired credential type obtained from the
+[azure-identity][azure_identity_credentials] library.
+
+To authenticate with AAD, you must first [pip][pip] install [`azure-identity`][azure_identity_pip] and
+[enable AAD authentication on your Purview resource][enable_aad]
+
+After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
+As an example, [DefaultAzureCredential][default_azure_credential]
+can be used to authenticate the client:
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+
+Use the returned token credential to authenticate the client:
+
+```python
+from azure.purview.account import PurviewAccountClient
+from azure.identity import DefaultAzureCredential
+
+credential = DefaultAzureCredential()
+client = PurviewAccountClient(endpoint="https://<my-account-name>.purview.azure.com", credential=credential)
+```
+
+## Key concepts
+
+### Client
+
+This package offers request builders so you can build http requests and send these requests to the service using the `send_request` method.
+For more information on how to use request builders and our clients, see [here][request_builders_and_client].
+
+## Examples
+
+The following section shows you how to initialize and authenticate your client, then list all of your keys.
+
+- [List All Keys](#list-all-keys "List All Keys")
+
+### List All Keys
+
+```python
+from azure.purview.account import PurviewAccountClient
+from azure.identity import DefaultAzureCredential
+from azure.purview.account.rest import accounts
+from azure.core.exceptions import HttpResponseError
+
+credential = DefaultAzureCredential()
+client = PurviewAccountClient(endpoint="https://<my-account-name>.purview.azure.com", credential=credential)
+
+request = accounts.build_list_keys_request()
+
+response = client.send_request(request)
+try:
+    response.raise_for_status()
+    assert response.status_code == 200
```

Comment:
I think this sample is out of date because we've embedded the requests. Can you switch this to calling `client.list_keys()`? Additionally, can you iterate over the returned keys and print out information on them? thanks!

Reply:
p.s., we also generally don't include `assert`s in our sample code, we use `assert` for testing instead